scripts/upgradeOriginalTokenBridgeProxy.js

```diff
@@ -1,14 +1,12 @@
 // scripts/deployOriginalTokenBridgeProxy.js
 const { ethers, upgrades } = require("hardhat")
 
-const OLD_PROXY_ADDRESS = "" // <-- Replace with the address of the proxy you want to upgrade
-
 async function main() {
-    const BridgeV2 = await ethers.getContractFactory("contracts/OriginalTokenBridgeUpgradableV2.sol:OriginalTokenBridgeUpgradableV2")
-    const bridge = await upgrades.upgradeProxy(OLD_PROXY_ADDRESS, BridgeV2)
+    const BridgeV2 = await ethers.getContractFactory("contracts/OriginalTokenBridgeUpgradable.sol:OriginalTokenBridgeUpgradable")
     console.log("Deploying bridge...")
-    await bridge.waitForDeployment()
-    console.log("Bridge deployed to:", await bridge.getAddress())
```

Comment:
why did you changed this?

Reply:
@leonprou beacuse earlier it was deploying the bridge and upgrading all in one tx, so we would need the pvt key of the owner, but it is owned by the safe, so instead this script only deploys the implementation so that we can upgrade using the safe

Reply:
got you 👌